infrastructure/postgres/store/features/update.go

```
@@ -0,0 +1,74 @@
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v5/pgtype"
+	"github.com/redcardinal-io/metering/domain/models"
+	"github.com/redcardinal-io/metering/domain/pkg/constants"
+	"github.com/redcardinal-io/metering/infrastructure/postgres"
+	"github.com/redcardinal-io/metering/infrastructure/postgres/gen"
+)
+
+func (p *PgFeatureRepository) UpdateFeatureByIDorSlug(ctx context.Context, idOrSlug string, input models.UpdateFeatureInput) (*models.Feature, error) {
+	tenantSlug := ctx.Value(constants.TenantSlugKey).(string)
+	// Try to parse as UUID first
+	parsedId, err := uuid.Parse(idOrSlug)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)